seed/lib/mcm/mapper.py

```diff
@@ -178,17 +177,57 @@ def _set_default_concat_config(concat):
     return concat
 
 
+def expand_field(field):
```

Comment:
new method to parse the field for delimiters. Note that the CSV module won't work because it appears to only work with 1 delimiter. I tried using the sniffer, but again, that finds the best delimiter for a single file/string, not all of them. So I just reverted to regular expressions.